content/dart/concepts/type-conversion/terms/toStringAsExponential/toStringAsExponential.md

```diff
@@ -0,0 +1,46 @@
+---
+Title: '.toStringAsExponential()'
+Description: 'Converts a number to its exponential notation in a string format.'
+Subjects:
+  - 'Computer Science'
+  - 'Mobile Development'
+  - 'Code Foundations'
+Tags:
+  - 'Dart'
+  - 'String'
+  - 'Method'
+CatalogContent:
+  - 'learn-dart'
+  - 'paths/computer-science'
+---
+
+The **`.toStringAsExponential()`** method is a Dart type conversion method that first converts a number to its exponential notation, then changes it to a string format. It can be used on integers and doubles.
```

Comment:
```suggestion
The **`.toStringAsExponential()`** method in Dart is used for type conversion, converting a number to exponential notation and then converting it to a string format. It can be applied to both integers and doubles.
```